conans/client/command.py

```diff
@@ -1226,11 +1226,11 @@ def main(args):
 
         def ctrl_c_handler(_, __):
             print('You pressed Ctrl+C!')
-            sys.exit(0)
+            sys.exit(-1)
```

Comment:
This will have different behavior in different systems, making it difficult to capture and process if anybody wants to do it. 

I suggest:

0: Success (done)
1: General ConanException error (done)
2: Migration error
3: Ctrl+C
4: Ctrl+Break

Reply:
@memsharded Yes you are totally right, I haven't seen any reference in docs about conan exit codes, shall I add it to docs too in a new section? if yes tell me where ...

Reply:
No, please do not document this yet. Lets wait for maturing and releasing the API.

Reply:
isn't it better to use 1 (EXIT_FAILURE) instead of -1?

Reply:
@memsharded @SSE4 fixed and added comment in docstring from main